app/networks/local-testnet/genesis.json

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
checked in local-testnet

Reply:
oops, must have added it before your .gitignore addition!